storage/src/test/java/org/apache/uniffle/storage/util/ShuffleStorageUtilsTest.java

```diff
@@ -251,6 +251,9 @@ public void getPartitionRangeTest() {
     int[] range = ShuffleStorageUtils.getPartitionRange(0, 1, 5);
     assertEquals(0, range[0]);
     assertEquals(0, range[1]);
+    range = ShuffleStorageUtils.getPartitionRange(5, 1, 5);
+    assertEquals(5, range[0]);
```

Comment:
@jerqi Saw your comment in the bottom thread. Just so that we are on the same page here, for `partitionId` >= `partitionNum`, is it supposed to return null then?